wls-gui-wahllokalsystem/src/components/stimmabgabevermerke/TheUWBStimmabgabevermerkeErfassenTable.vue

```diff
@@ -107,17 +107,31 @@
     <base-dialog
       :visible="isDeleteDialogVisible"
       dialogtitle="Reduzierung der Blätteranzahl des Wählerverzeichnisses"
-      confirmtext="Trotzdem Löschen"
-      canceltext="Abbrechen"
+      confirmtext="Hinweis schließen"
       icon="$information"
-      @cancel="isDeleteDialogVisible = false"
-      @confirm="onDialogConfirmDeletingRows"
-      ><div>
-        Sie wollen Blätter löschen, für die Sie Stimmabgabevermerke eingetragen
-        haben. Wenn Sie diese löschen, werden dadurch auch die Werte für die
-        Stimmabgabevermerke gelöscht.
-      </div></base-dialog
-    >
+      @confirm="isDeleteDialogVisible = false"
+      ><div class="mb-4">
+        Sie haben die Blätteranzahl des Wählerverzeichnisses verändert. Zeilen
+        können nur gelöscht werden, wenn noch keine Stimmabgabevermerke erfasst
+        wurden.
+      </div>
+      <div>
+        {{ blattnummernThatPreventDeletion.length }}
+        {{
+          blattnummernThatPreventDeletion.length === 1
+            ? "Element verhindert"
+            : "Elemente verhindern"
+        }}
+        das Löschen: Für die
+        {{
+          blattnummernThatPreventDeletion.length === 1
+            ? "Blattnummer"
+            : "Blattnummern"
+        }}
```

Comment:
die zwei parts würde ich zusammenfassen, damit der code leichter lesbar ist.

```suggestion
        {{
          blattnummernThatPreventDeletion.length === 1
            ? "Element verhindert das Löschen: Für die Blattnummer"
            : "Elemente verhindern das Löschen: Für die Blattnummern"
        }}
```

Reply:
generell wäre es ggf auch ne option die generierung des gesamten texts in den script teil zu verlegen, weil da ja logik mit drin ist

Reply:
[da690e8](https://github.com/it-at-m/Wahllokalsystem/pull/2005/commits/da690e8b81068d1c04aa041f0740f575e920b821)